src/Menu.java

```diff
@@ -1,12 +1,30 @@
+import java.io.*;
+import java.util.ArrayList;
 
 public class Menu {
```

Comment:
### Utility classes or classes that have only static methods  should not have public/Default constructors as there should be no need to ever create an Object of this class.So declare a private constructor like : 

`	private Menu() {
		throw new UnsupportedOperationException(Menu.class.getName());
	}`